dataphos-persistor/values.yaml

```diff
@@ -1,10 +1,185 @@
+  ## @param PES_ID.storage The reference to the storage messages will be persisted to.
+  ## @param PES_ID.indexer The reference to the indexer the Persistor is tied to. If not set, the Indexer plugin is disabled.
+  ## @param PES_ID.deadletterTopic The name of the dead-letter topic used for failed data processing. If not set, dead-lettering is disabled (must be enabled if kafka broker is used).
+  ## @param PES_ID.replicas The number of replicas of a given Persistor instance to pull/process messages simultaneously.
+  ## @param PES_ID.storageCustomValues Comma-separated list of key:value pairs to include in the file path. If the file path contains key1:value1 and STORAGE_MASK contains key1, then the path will contain value1.
+  ## @param PES_ID.storageMask Structure of the path under which batches are stored.
+  ## @param RSMB_ID.clientID The Client ID of the Azure Service Principal with the required role assignments.
+  ## @param RSMB_ID.clientSecret The Client Secret of the Azure Service Principal with the required role assignments.
+  ## @param RSMB_ID.tenantID The Tenant ID of the Azure Service Principal with the required role assignments.
+  ##
+  absPes:
+    broker: someKafkaBroker
+    topic: input-topic
+    consumerID: input-topic-sub
+    storage: someGCSStorage
+    storageTargetID:
+    indexer: kafkaIndexer
+    deadletterTopic: dl-topic
+    replicas: 1
+    storageCustomValues: "project:dataphos"
+    storageMask: "project/product/year/month/day/hour"
+    clientID:
+    clientSecret:
+    tenantID:
+  ## @param IDX_ID.batchSize Maximum number of messages in a batch.
+  ## @param IDX_ID.batchMemory Maximum bytes in batch.
+  ## @param IDX_ID.batchTimeout Time to wait before writing batch if it is not full.
+  ## @param IDX_ID.serviceAccountSecret The reference to a secret that contains a key.json key and the contents of a Google Service Account JSON file as its contents.
+  ##
+  gcsPes:
+    broker: somePubSubBroker
+    pullMode: batch
+    topic: input-topic-2
+    consumerID: input-topic-sub-2
+    storage: someGCSStorage
+    storageTargetID: bucket1
+    deadletterBroker: somePubSubBroker
+    deadletterTopic: dl-topic-2
+    batchSize: 5000
+    batchMemory: 1000000
+    batchTimeout: 30s
+    serviceAccountSecret:
+    replicas: 2
+
+## The object containing the general information on the brokers the validator service will want to associate with.
+##
+brokers:
+  ## The object representing an individual broker's configuration.
+  ## @param BROKER_ID.type Denotes the broker's type.
+  ## @param BROKER_ID.brokerAddr The Kafka bootstrap server address.
+  ##
+  someKafkaBroker:
+    type: kafka
+    brokerAddr:
+  ## @param BROKER_ID.projectID The GCP project ID.
+  ##
+  somePubSubBroker:
+    type: pubsub
+    projectID:
+  ## @param BROKER_ID.connectionString The Azure Service Bus Namespace connection string.
+  ##
+  someServiceBusBroker:
+    type: servicebus
+    connectionString:
+
```

Comment:
Comment these out, leaving only Kafka not commented.
Also, it's maybe not a bad idea to add a comment "In case you use PubSub, comment out this section" or something similar. Same for ServiceBus.